hack/create-kubeconfig.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Authors of Arktos.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+LOG_DIR=${LOG_DIR:-"/tmp"}
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+CERT_DIR=${CERT_DIR:-"/var/run/kubernetes"}
+CERT_ROOTNAME=${CERT_ROOTNAME:-"host"}
+
+source "${KUBE_ROOT}/hack/lib/util.sh"
+
+# Ensure CERT_DIR is created for crt/key and kubeconfig
+mkdir -p "${CERT_DIR}" &>/dev/null || sudo mkdir -p "${CERT_DIR}"
+CONTROLPLANE_SUDO=$(test -w "${CERT_DIR}" || echo "sudo -E")
+
+# There are following commands to run the script
+# hack/create-kubeconfig.sh is to create a kubeconfig giving the following params
+# hack/create-kubeconfig.sh extract key is to extract content from an existing kubeconfig.
+if [ $# -lt 1 ] ; then
```

Comment:
This looks straight forward. Just to help others to ramp up, please add detailed manual once all the steps are flushed out. Thanks!